data/data/powervs/loadbalancer/alb.tf

```diff
@@ -0,0 +1,151 @@
+################################################################
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Licensed Materials - Property of IBM
+#
+# ©Copyright IBM Corp. 2021
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################
+
+locals {
+  api_servers       = concat([var.bootstrap_ip], var.master_ips)
+  api_servers_count = 4 # bootstrap + master 
+  app_servers       = var.master_ips
+  app_servers_count = 3
```

Comment:
I see these counts are hardcoded, I'm wondering whether we can just derive them from the count of var.bootstrap_ip, var.master_ips

Reply:
Yeah we can derive from the vars. Wondering if it'll ever change ?

Reply:
default value if 3 but there is scope of changing this value via `controlPlane.replicas` param

Reply:
depends on OpenShift allowing different number of masters :-) 
I'll change it derive the count from master.ips.